bindings/rust/aws-lc-fips-sys-template/README.md

```diff
@@ -0,0 +1,44 @@
+# aws-lc-fips-sys
+
+[![crates.io](https://img.shields.io/crates/v/aws-lc-fips-sys.svg)](https://crates.io/crates/aws-lc-fips-sys)
+
+Low level AWS-LC FIPS bindings for the Rust programming language. We do not recommend directly relying on these bindings.
+
+[Documentation](https://github.com/awslabs/aws-lc).
+
+## Release Support
+
+This crate pulls in the source code of the latest AWS-LC FIPS branch to build with it. Bindings for platforms we officially support are pre-generated.
+The platforms which `aws-lc-fips-sys` builds on is limited to the platforms where the AWS-LC FIPS static build is supported.
+
+### Pregenerated Bindings Availability
+
+CPU|OS
+-------------|-------------
+x86-64|Ubuntu
+arm-64|Ubuntu
+
+### Tested AWS-LC FIPS Build Environments
+
+Compiler|CPU|OS
+-------------|-------------|-------------
+gcc-7|x86-64|Ubuntu/AL2
+clang-7|x86-64|Ubuntu/AL2
+clang-7|arm-64|Ubuntu/AL2
+gcc 4.8.5|x86-64| CentOS 7
```

Comment:
Instead of listing the FIPS tested environments here, should we link-to/reference documentation provided elsewhere for AWS-LC FIPS.  If we list them here, we'll need to ensure we keep the list in sync with our CI/documentation elsewhere.

Reply:
Sure, the FIPS tested environments for the static build are limited, so I've used the opportunity to add this in `tests/ci/README.md`. I've linked to the new section I added from here.